src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/AboutSubcommand.java

```diff
@@ -38,12 +45,41 @@ public Integer call() throws IOException, SQLException {
         "This tool allows to extract meta information from a data warehouse that allows");
     System.out.println("to make an assessment for migration.");
     System.out.println();
-    try (InputStreamReader reader =
-        new InputStreamReader(this.getClass().getResource("/NOTICE.txt").openStream())) {
-      for (String s : CharStreams.readLines(reader)) {
-        System.out.println(s);
+    System.out.println("Data Warehouse Assessment Extraction Client");
+    System.out.println("Copyright 2021 Google LLC");
+    System.out.println();
+    List<String> dependencyNames =
+        Resources.readLines(
+            AboutSubcommand.class.getResource("/third_party/dependency-list.txt"), UTF_8);
```

Comment:
Consider adding Objects.requireNonNull to check the input to Resources.readlines(). Same for other occurrences.

Reply:
I've replaced with `Resources.getResources` that performs the null check. I've also added `AboutSubcommandTest` to make sure no exception is thrown during execution of "about" command.